src/index.md

```diff
@@ -126,11 +127,15 @@ The topology information is stored as attributes to a dummy variable (in the exa
 | boundary_node_connectivity |
 | face_coordinates |
 | edge_coordinates |
+| edge_dimension |
 
 *The "Optionally required" attribute `edge_node_connectivity` is required only if you want to store data on the edges (i.e. if you mind the numbering order of the edges).
 
 The attribute `topology_dimension` indicates the highest dimensionality of the geometric elements; for a 2-dimensional (triangular) mesh this should be 2. The attribute `node_coordinates` points to the auxiliary coordinate variables representing the node locations (latitude, longitude, and optional elevation or other coordinates). These auxiliary coordinate variables will have length nNodes. The attribute `face_node_connectivity` points to an index variable identifying for every face (here consistently triangle) the indices of its three corner nodes. The corner nodes should be specified in anticlockwise (also referred to as counterclockwise) direction as viewed from above (consistent with the CF-convention for `bounds` of p-sided cells. The connectivity array will thus be a matrix of size nFaces x 3. For the indexing one may use either 0- or 1-based indexing; the convention used should be specified using a `start_index` attribute to the index variable (i.e. Mesh2_face_nodes in the example below). Consistent with the CF-conventions [compression](http://cf-convention.github.io/1.4.html#compression-by-gathering) option, the connectivity indices are 0-based by default. See [this section on zero or one-based indexing](#zero-or-one) for more details.
 
+The `face_dimension` attribute specifies with netcdf dimension is used to indicate the index of the face. This is needed because some applications store the data with the fastest varying index first, and some with that index last. The default is to use a (num_faces, 3) array for triangles, but some applications might use a (3, num_faces) order, in which case the `face_dimension` attribute can help the client code disambiguate. The `edge_dimension` attribute is similar for the edge_coordinates array.
```

Comment:
@ChrisBarker-NOAA , how about modifying "specifies with netcdf dimension" to "specified which dimension" ?